doc/manual/17-configuration.en.md

```diff
@@ -6,14 +6,19 @@ There are two types of configuration files in Barman:
 
 - **global/general configuration**
 - **server configuration**
+- **model configuration**
 
 The main configuration file (set to `/etc/barman.conf` by default) contains general options such as main directory, system user, log file, and so on.
 
 Server configuration files, one for each server to be backed up by Barman, are located in the `/etc/barman.d` directory and must have a `.conf` suffix.
 
+Similarly, model configuration files, one for each model that can be applied to a server which is part of a cluster, are located in the `/etc/barman.d` directory and must have a `.conf` suffix.
+
+> *NOTE*: models define a set of configuration overrides which can be applied on top of the configuration of Barman servers that are part of the same cluster as the model, through the `barman config-switch` command.
```

Comment:
Could we link to the `barman config-switch` section here?

Reply:
I think that makes sense, I'll take a look at adding a ref.

Reply:
Applied.